objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/ObjectRecognitionConfigHandler.java

```diff
@@ -323,11 +326,18 @@ boolean createImageRetriever(Map<String, ?> dataSourceConfig) {
     private boolean prepareCommunication(Map<String, ?> general) {
         int polling = -1; // initializing to an invalid input
         boolean queryable = false;
+        source.pool = new ThreadPoolExecutor(5, 5, 0l, TimeUnit.MILLISECONDS, new LinkedBlockingQueue<Runnable>(10));
```

Comment:
I think it would be good, before we push to master, to have the a configurable `concurrentAnalysisCount` and `queuedAnalysisCount` (to be the 5 & 10, respectively).  That said, assuming other things are OK, we can publish the branch to Brett & company and let them play with it as is.

That shouldn't be very hard to do.  I would suggest that we support `concurrentAnalysisCount (defaults to 5), and `queuedAnalysisCount` (defaults to 2 x `concurrentAnalysisCount`). We will note (in the README) that each of these increases memory pressure (the total of the two images in memory at a time) and CPU/gCPU cost).